include/quicr/quicr_namespace.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "quicr_name.h"
+
+namespace quicr
+{
+class Namespace
+{
+public:
+    Namespace() = delete;
+    Namespace(const Name& name);
+    Namespace(Name&& name);
+
+    bool contains(const Name& name) const;
+    bool contains(const Namespace& name_space) const;
+
+    friend bool operator==(const Namespace& a, const Namespace& b);
+    friend bool operator!=(const Namespace& a, const Namespace& b);
+    friend bool operator>(const Namespace& a, const Namespace& b);
+    friend bool operator<(const Namespace& a, const Namespace& b);
+
+private:
+    Name _name;
+    uint64_t _sig_bits{120};
```

Comment:
don't default this and I think given the valid range is 0 to 127 , probably should just be uint16_t